app/renderer/lib/appium-client.js

```diff
@@ -292,6 +306,55 @@ export default class AppiumClient {
     // If the app under test returns non JSON format response
     return false;
   }
+
+  /**
+   * Parse the Android contexts webview data
+   *
+   * Returns
+   * {
+   *   id: string;             // NATIVE_APP or the webview name
+   *   title?: string;         // the name of the page
+   *   url?: string;           // the url
+   *   bundleId?: string;      // for iOS
+   *   packageName?: string;   // for Android
+   *   handle?: string;        // the id of the active page in the webview of Android
+   * }[];
+   */
+  parseAndroidContexts (contexts) {
```

Comment:
it would be useful to have a unit test for this helper

Reply:
Done